README.md

```diff
@@ -1,12 +1,31 @@
-# openSUSE and Tini image
+# openSUSE Base image
 
-This project builds an openSUSE-based image which includes Tini. It can be used as a base image by projects which require Tini.
+This project builds an openSUSE-based image which includes Tini, cURL and the PostgreSQL client.
 
 ### Tini
 [Tini](https://github.com/krallin/tini) is pre-installed in the container.  If the image entrypoint is not overwritten then it will be automatically used.
 
+### PostgreSQL Client
+[PostgreSQL](https://wiki.postgresql.org/wiki/Main_Page) Client is pre-installed in the container. This enables projects to automatically install required PostgreSQL databases if required.
+
 ### Startup Scripts
 Any executable scripts added to the `/startup/startup.d/` directory will be automatically run each time the container is started (assuming the image entrypoint is not overwritten).
 
+#### Database Check and Create Script
+The `check-create-pgdb.sh` script is provided in the `/startup/startup.d/` directory by default. This script enables consumers of this image to define a PostgreSQL database that will be created during startup of the consuming container. The script will also check if the database already exists and if so will not attempt to create it.
```

Comment:
"is provided in the `/startup/startup.d/` directory by default" --> "comes pre-installed"